bip-0009.mediawiki

```diff
@@ -23,6 +28,11 @@ BIP 34 introduced a mechanism for doing soft-forking changes without a predefine
 
 In addition, BIP 34 made the integer comparison (nVersion >= 2) a consensus rule after its 95% threshold was reached, removing 2<sup>31</sup>+2 values from the set of valid version numbers (all negative numbers, as nVersion is interpreted as a signed integer, as well as 0 and 1). This indicates another downside this approach: every upgrade permanently restricts the set of allowed nVersion field values. This approach was later reused in BIP 66 and BIP 65, which further removed nVersions 2 and 3 as valid options. As will be shown further, this is unnecessary.
 
+==2021 update==
+
+* '''min_activation_height''' was added. Clients unaware of softforks introduced after SegWit can ignore this in the context of their upgrade warnings.
+* removed state transition from '''defined''' to '''failed'''. This change is safe to ignore for all (burried) softforks deployed before 2021.
```

Comment:
BIP-wide: use either "soft fork" or "softfork", but maybe not both

Reply:
Or soft-fork? It's already inconsistent. I might just add a commit that picks one.

Reply:
When in doubt, I consult https://github.com/bitcoinops/bitcoinops.github.io/blob/master/STYLE.md (which says to use "soft fork" and not "softfork")